packages/client/hooks/useAnimatedPhaseListChildren.ts

```diff
@@ -0,0 +1,25 @@
+import {useEffect, useRef, useState} from 'react'
+
+const useAnimatedPhaseListChildren = (isActive: boolean, itemCount: number) => {
+  const [height, setHeight] = useState(isActive ? 'fit-content' : 0)
+  const ref = useRef<HTMLDivElement>(null)
+  useEffect(() => {
+    const scrollHeight = ref.current?.scrollHeight
+    setHeight(scrollHeight ?? 0)
+  }, [itemCount])
+
+  const wasActiveRef = useRef(isActive)
+  useEffect(() => {
+    if (isActive === wasActiveRef.current) return
+    if (isActive) {
+      ref.current!.style.overflow = 'hidden'
+      setTimeout(() => {
+        ref.current!.style.overflow = ''
+      }, 300)
```

Comment:
I have two minor suggestions
1. It'd be nice to clear the timeout in effect cleanup just to avoid some edge cases crashing, especially considering we're assuming the ref is always there 
2. Do we want to extract `300` as some kind of const to use in similar scenarios? Naming this would also be helpful in understanding why do we need the timeout in the first place 🙏

Reply:
Thanks for the review! Makes sense - I've just pushed an update 👍